site_scons/site_tools/go_builder.py

```diff
@@ -81,6 +91,10 @@ def _check_go_version(context):
             context.Result(0)
             return 0
 
+        context.Display(f'Getting minimum {env.d_go_bin} version... ')
```

Comment:
You're not checking the minimum version of the installed binary, you're checking the binary against a minimum version of the language.
```suggestion
        context.Display('Getting minimum go version... ')
```
As this code stands it's outputting the below which feels incorrect.
```
Checking for Go compiler... /bin/go
Getting minimum /bin/go version... 1.21
Checking /bin/go version... 1.21.13
```

Reply:
Fair point, fixing.